static/js/theme-map/Maps/Providers/Google.js

```diff
@@ -80,6 +80,9 @@ class GoogleMap extends ProviderMap {
     google.maps.event.addListener(this.map, 'click', () => {
       this._canvasClickHandler();
     });
+    google.maps.event.addListenerOnce(this.map, 'tilesloaded', () => {
```

Comment:
is there a reason we only want to listen to the `tilesloaded` event only once? I assume every new search would re-load the tiles and trigger this event, do we not want to execute `loadHandler` function those times?

Reply:
the load handler is only meant to run once when the map loads